.github/workflows/test_build.yml

```diff
@@ -117,8 +117,8 @@ jobs:
         run: |
           cmake --build . --target build-libraries  -j4
           cmake --build . --target build-programs   -j4
+          cmake --build . --target M2-core          -j4
```

Comment:
`M2-core` is a dependency of `install-packages`, so this is unnecessary.

Reply:
Also, again, the `-j4` are unnecessary :)

Reply:
* I like having `M2-core` there explicitly because it's informational.
* I can believe that the `e/` code could not benefit from parallelism. Not sure about `{c,d}/`. But `build-libraries` and `build-programs` are trivially parallelizable, aren't they? (May be with minor exceptions.)

Reply:
For the `-j4`, I agree Ninja doesn't need it &mdash; it defaults to `(num cores)+2`, which is `2+2=4` for the virtual machines running the builds. I can remove it next time with the next iteration.

Reply:
Weird, my previous message logged as a global comment rather than to this. I'll quote it for the record:
> Everything is parallelizable, and all of it is parallelized automatically
by Ninja. That argument is just extra.